pandas/compat/numpy/__init__.py

```diff
@@ -5,6 +5,8 @@
 
 import numpy as np
 
+from pandas.core.dtypes.common import is_extension_array_dtype
```

Comment:
if there's any viable way to avoid this, we'd rather not import from pandas.core here